src/containers/Dashboard/Tutorials/Tutorial.js

```diff
@@ -1,14 +1,54 @@
 import React, { Component, Fragment } from "react";
-import {
-  INTERVIEW_PREP_KIT,
-  STATISTICS,
-  JAVASCRIPT,
-  CODE
-} from "../../../constants";
+// import {
+//   INTERVIEW_PREP_KIT
+// } from "../../../constants";
 import "./Tutorial.scss";
 import TutorialCard from "./TutorialCard";
+import { addNotification } from "../../../utilities";
+import API from "../../../api/index";
 
 class Card extends Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      tutorials: [],
+      tutorial : []
+    };
+  }
+
+  componentWillMount() {
+    API.getTutorials(result => {
+      if (result.status === "200") {
+        this.setState({
+          tutorials: result.data
+        });
+        if (this.state.tutorials.length < 6) {
+          this.state.tutorials.map((tutorial, index) => {
+            this.setState({
```

Comment:
@HarisShahidHameed This is not how you set state in react. `setState` is an async fucntion so you cannot expect to change the state depending upon the existing value of state in a loop.

Reply:
Done.